components/brave_new_tab_ui/storage.ts

```diff
@@ -4,13 +4,18 @@
 
 // Utils
 import { debounce } from '../common/debounce'
+import {
+  getLoadTimeDataString,
+  getLoadTimeDataBoolean,
+  getChromeDataBoolean
```

Comment:
These methods are just wrappers around chrome/custom window APIs since Storybook doesn't have access to them.

Reply:
When Storybook runs through these files it loads the default state and throws an error since `chrome` and `window.loadTimeData` are not available. The alternative I thought would be mocking default state which is what I tried to avoid. I'm ok any other decision just thought this one was best

Reply:
Are you saying it's an import timing error because the _store_ JS is parsed before _.storybook/config.js_ is? Seems to be working ok with whatever else is calling window.loadTimeData.getString which was the cause for us to put that in to window.loadTimeData in the first place.

Reply:
Nevermind, I got what you mean now. Added other mocks to the config file per your suggestion.